package.json

```diff
@@ -18,17 +18,19 @@
     "Raul Ochoa <rochoa@cartodb.com>",
     "Nicklas Gummesson <nicklas@cartodb.com>",
     "Pablo Alonso <pablo@cartodb.com>",
-    "Antonio Santiago Pérez <asantiago@cartodb.com>"
+    "Antonio Santiago Pérez <asantiago@cartodb.com>",
+    "Carlos Matallín <matallo@carto.com>"
```

Comment:
... come on! let's remove the contributors please! :joy:
It appears in the Github tab.